temporalcli/commands.taskqueue_update_build_ids.go

```diff
@@ -0,0 +1,80 @@
+package temporalcli
+
+import (
+	"fmt"
+
+	"go.temporal.io/sdk/client"
+)
+
+func updateBuildIds(cctx *CommandContext, parent *TemporalTaskQueueCommand, options *client.UpdateWorkerBuildIdCompatibilityOptions) error {
+	cl, err := parent.ClientOptions.dialClient(cctx)
```

Comment:
```suggestion
func (c *TemporalTaskQueueUpdateBuildIdsCommand) updateBuildIds(cctx *CommandContext, options *client.UpdateWorkerBuildIdCompatibilityOptions) error {
	cl, err := c.Parent.ClientOptions.dialClient(cctx)
```

Can make the common piece the receiver if you wanted to just `c.Parent.updateBuildIds(cctx, options)` in other commands below without having a top-level function. But not required or anything.

Reply:
Didn't realize I had a base command build by the parser... That's great thanks!